core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.spec.ts

```diff
@@ -655,6 +657,49 @@ describe('Translation Modal Component', () => {
       });
     });
 
+    describe('when some elements are hidden', () => {
+      it('should submit the translation if it is valid', () => {
+        component.textToTranslate = '<p>James and Uncle Berry were in the ' +
```

Comment:
It looks like the difference in the text to translate is the image element at the end. Can you explain in a comment how this replicates a "hidden" element, and what hidden means? I know we already described the situation in the implementation file, but it would be good to also briefly explain it here in the context of the test.